fakeDataGenerator.js

```diff
@@ -0,0 +1,169 @@
+const faker = require('faker');
+const fs = require('fs');
+const moment = require('moment');
+
+faker.seed(105);
+
+const fakeRestaurants = [];
+
+for (let i = 1; i <= 100; i++) {
+  const fakeRestaurant = {id: i};
+  const fakeWords = [];
+  const length = faker.random.number() % 3;
+
+  for (let i = 0; i <= length; i++) {
+    fakeWords.push(faker.random.word());
+  }
+
+  fakeRestaurant.name = fakeWords.join(' ');
+
+  fakeRestaurants.push(fakeRestaurant);
+}
+
+const fakeUsers = [];
+
+for (let i = 1; i <= 100; i++) {
+  const fakeUser = {id: i};
+  fakeUser.name = faker.name.findName();
+  fakeUsers.push(fakeUser);
+}
+
+
+const fakeReviewTextGenerator = () => {
+  const reviewText = [];
+  const paragraphs = faker.random.number(4);
+  for (let p = 0; p <= paragraphs; p++) {
+    let paragraph = '';
+    const sentences = faker.random.number(3) + 2;
+    for (let s = 0; s <= sentences; s++) {
+      paragraph += faker.lorem.sentence();
```

Comment:
Consider faker.lorem.paragraph(), although they should be functionally the same, unless you're shooting for exactly 2-5 sentences. I'm not sure faker.lorem.paragraph() can be passed some number of sentences.

Reply:
I do want some 2 -sentence paras, but I think you're right and might be able to do the same with faker.lorem.paragraph. Will investigate.